lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/test/RoundTripTest.java

```diff
@@ -48,7 +49,8 @@ public void testRoundTrip() throws Exception {
   public void testRoundTripWithClassHierarchy() throws Exception {
     // TODO: use 'getConfiguration' instead of 'getConfigurationWithoutList' after #192 is fixed
     final Configuration confBefore = ObjectTreeTest.getConfigurationWithoutList();
-    final ClassHierarchy c = new ProtocolBufferClassHierarchy(ProtocolBufferClassHierarchy.serialize(confBefore.getClassHierarchy()));
+    final ClassHierarchySerializer chSerializer = new AvroClassHierarchySerializer();
+    final ClassHierarchy c = chSerializer.fromString(chSerializer.toString(confBefore.getClassHierarchy()));
```

Comment:
Same thing, really. But we do go down all the way to `byte[]` then, which would be the common use case. Maybe we should do both? We do that in `org.apache.reef.tang.test.RoundTripTest` and its implementations.